src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 800px;
+  padding: 10px;
+
+  @extend %start-day-sat;
```

Comment:
Ensure that the placeholder `%start-day-sat` is defined in your SCSS files. If it's not defined, this will cause a compilation error.

Reply:
its in another file and import on index.scss